eyes-tilt-motor-sizing/README.md

```diff
@@ -24,44 +33,131 @@ The complete model is divided into the following parts:
 
 ## 3.2. Mechanics
 
-The mechanics part of the model is generated directly from the Creo CAD.
+The mechanics part of the model is generated directly from the Creo CAD;
+and then we imported it into SimScape. 
+To generate it, execute Matlab and then on the command window:
+```matlab
+cd study-improve-head-edpr\eyes-tilt-motor-sizing
+smimport('models/resources_simscape/ICUB3_020_G_300.xml');
+```
+See also the official [import procedure](https://it.mathworks.com/help/physmod/sm/ug/import-a-cad-assembly-model.html#:~:text=You%20import%20a%20CAD%20model,body%20geometries%20for%20visualization%20purposes).
 
 <img src="assets/mech001.png" alt="drawing" width="900"/>
 
-Some minor fix should be done after the Creo generation:
+Some minor fixes should be done after the Creo generation:
 - Remove SixDof
 - Check gravity direction
-- Add the out position, velocity port and add the in torque port  <br><br>
-<img src="assets/mech003.png" alt="drawing" width="300"/>   <br><br>
+- Add the out position, velocity port and add the in torque port  
+
+<br><br>
+<img src="assets/mech003.png" alt="drawing" width="300"/>  
+<br><br>
 <img src="assets/mech004.png" alt="drawing" width="300"/>    
 - Manage the position offset  
-  <img src="assets/mech002.png" alt="drawing" width="300"/>
+  
+<img src="assets/mech002.png" alt="drawing" width="500"/>  
+
+- Add the out ports to the model (Tilt, Vel) and the in port (Torque).
 
 ## 3.3. Control
 
 The control block is shown in the following figure:  
 
-<img src="assets/pid001.png" alt="drawing" width="800"/>  
+<img src="assets/pid001.png" alt="drawing" width="600"/>  
+
+It is a discrete PI.  
+Configured parameters:
+|Name|Value|
+|-|-|
+|Proportional|10|
+|Integral|10|
```

Comment:
```suggestion
```

This can be removed. It's not very relevant to report the actual values of the gains in the documentation.
Gains can be easily retrieved straight from the model instead so that the documentation hasn't to be maintained.